src/Redux/Store/Expeditions/Expeditions/sideEffects/helpers.ts

```diff
@@ -177,7 +177,7 @@ export const getMageIds = ({
   if (mage === undefined) {
     return baseResult
   } else {
-    const ids = mage.ids.filter(idOrRandom => typeof idOrRandom === 'string')
+    const ids = mage.ids.filter((idOrRandom) => typeof idOrRandom === 'string')
 
     const randomTreasureConfigs = mage.ids.filter(
```

Comment:
we should definitely change the naming here - this is probably a copy and paste issue (i copied some internals from the treasure generation), but its very misleading. -> in turn we should also change it in your code as well.